scripts/release-merge-to-main.ts

```diff
@@ -77,11 +80,15 @@ async function ensureBranch(owner: string, repo: string, branchName: string, tag
       await git.updateRef({
         owner, repo, ref, sha,
       });
+    } else {
+      console.log(`Branch ${ owner }/${ repo }/${ ref } is already up-to-date.`);
     }
   } catch (ex) {
     if (!(ex instanceof RequestError) || ex.status !== 404) {
       throw ex;
     }
+    console.log(`Creating new branch ${ owner }/${ repo }/${ ref }` +
+      `at ${ sha }`);
```

Comment:
Given how long other lines below are, I don't understand why this one had to be split up.

```suggestion
    console.log(`Creating new branch ${ owner }/${ repo }/${ ref } at ${ sha }`);
```